src/Processor/Gotenberg.php

```diff
@@ -106,6 +106,18 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
         $tempFileName = uniqid('web2print_');
 
         $chromium = GotenbergAPI::chromium(\Pimcore\Config::getSystemConfiguration('gotenberg')['base_url']);
+        // To support gotenberg-php v2 and so on
+        if (method_exists($chromium, 'pdf')) {
+            $chromium = $chromium->pdf();
+        } else {
+            // gotenberg-php v1 BC Layer for unsupported methods in v2
+            if (isset($params['userAgent'])) {
+               $chromium->userAgent($params['userAgent']);
```

Comment:
PHPStan is complaining, should 
```suggestion
               /* @phpstan-ignore-next-line */
               $chromium->userAgent($params['userAgent']);
```
do the trick or is there a better way to deal with it?

Reply:
Well, in that case it´s the easeiest solution i guess. Just to get PHPStan green. The code itself isn´t wrong.

Alternative we re-implement ``` && method_exists($chromium, 'userAgent')``` - that would do the same and remove warnings in IDE too. Personally i prefer last solution approach, because it´s clear and safe. 

What´s the internal Pimcore Codeing Standard on that point? @kingjia90

Reply:
There's no internal coding standard as far as i know 😄 
Then let's re-implement the `method_extsts`, thank you again